voidcrew/modules/overmap/code/modules/overmap/ship.dm

```diff
@@ -190,6 +192,16 @@
 //	update_docked_bools() //voidcrew todo: ship functionality
 	qdel(src)
 
+/obj/structure/overmap/ship/proc/ship_announce(message, title, must_be_same_z_level = FALSE, sound)
```

Comment:
Leaving as a note for a future PR, ships should use a signal on a captain joining through this